static-checks/ansible/allowed-modules/get_all_ansible_modules.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```

Comment:
I believe this could be rewritten to Python3. Thus avoiding Bash script there and making it even nicer - getting rid of multiple pipes that are thee times there:
```
for package in ['ansible', 'ansible-core', 'rhc-worker-playbook']:
    ...
```
similarly for `else` part.

Reply:
Makes sense, I will look into it.